### PR TITLE
Add Nutrition Facts for products

### PR DESCRIPTION
**Pull request by tien4112004**
This pull request focused on nutrition facts calculation in the server-side.

### Controller and Service Enhancements:
* Modified `UpdatePeopleCount` method in `ClientOrderController` to use `UpdateOrderPeopleCountDto` instead of a query parameter.
* Added `IOrderService` to `KitchenOrderController` and updated the constructor accordingly.
* Updated `UpdateOrderItemsStatus` method in `KitchenOrderController` to calculate the total cancel amount.

### Repository and Configuration Updates:
* Added `INutritionFactRepository` to `AppServiceConfig` and `IUnitOfWork`. 
* Created `INutritionFactRepository` interface.

### Data Seeding:
* Added new data entries to CSV files for categories, identity roles, images, nutrition facts, order items, orders, products, tables, and users.

### CODE REVIEW

File: BistroQ.Infrastructure/Data/nutritionfacts.csv
Comment (Tondeptrai23) [inline BistroQ.Infrastructure/Data/nutritionfacts.csv:1]: Please fix this to correct values before merging

**Review by Tondeptrai23 — APPROVED** [review]

File: BistroQ.Tests/ClientOrderControllerTest.cs [hunk 1]
```diff
@@ -203,7 +204,11 @@ public async Task UpdatePeopleCount_Success()
 		_mockOrderService.Setup(x => x.UpdatePeopleCount(1, 5)).ReturnsAsync(order);
 
 		// Act
-		var result = await _clientOrderController.UpdatePeopleCount(5) as OkObjectResult;
+		var request = new UpdateOrderPeopleCountDto
+		{
+			PeopleCount = 5
```
Comment (Tondeptrai23) [inline BistroQ.Tests/ClientOrderControllerTest.cs:209]: Why is this set to a fixed value?

File: BistroQ.Services/Services/OrderService.cs [hunk 2]
```diff
@@ -60,10 +61,33 @@ public async Task<DetailOrderDto> GetOrder(int tableId)
...
 			throw new ResourceNotFoundException("Order not found");
 		}
-
+		
 		order.OrderItems = _mapper.Map<List<OrderItem>>(order.OrderItems);
 
-		return _mapper.Map<DetailOrderDto>(order);
+		var result = _mapper.Map<DetailOrderDto>(order);
+		
+#region braindead-code
```
Comment (Tondeptrai23) [inline BistroQ.Services/Services/OrderService.cs:69]: Add a `GetByIdsAsync` method to `INutritionRepository` can resolve this

File: BistroQ.Tests/ClientOrderControllerTest.cs (hunk 1 above)
Comment (tien4112004) [inline BistroQ.Tests/ClientOrderControllerTest.cs:209]: Bro, this is a test...

File: BistroQ.Services/Services/OrderService.cs (hunk 2 above)
Comment (tien4112004) [reply BistroQ.Services/Services/OrderService.cs:69]: Done.

**Pull request merged**